web/src/components/forms/validator-form.tsx

```diff
@@ -405,17 +407,24 @@ export const ValidatorForm: FC = () => {
               <>
                 <div className="alert alert-danger" role="alert">
                   <p className="mb-0">PEP is invalid!</p>
-                  <p className="mb-0">Errors:</p>
+                  <p className="mb-0">
+                    Errors found in {result.error_type}{' '}
+                    {result.sample_names && (
+                      <OverlayTrigger overlay={<Tooltip id="validation">{result.sample_names}</Tooltip>}>
```

Comment:
Does the tooltip render nicely? Do you have a screenshot? I guess I could pull. Curious what `result.sample_names` looks like when thrown in a `div` raw.